src/features/settings/utils/useFormattedDate.ts

```diff
@@ -0,0 +1,8 @@
+export const useFormattedDate = (date: string | undefined) => {
```

Comment:
super minor but I prefer `date?: string`

Reply:
Fixed. This is less verbose and nice looking